flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/CliClient.java

```diff
@@ -613,6 +624,20 @@ private void printInfo(String message) {
 		terminal.flush();
 	}
 
+	/**
+	 * Output Results in non-interactive way.
+	 *
+	 * @return
+	 */
+	void printRows(final ResultDescriptor desc, long start, long end) {
+		final String msg = "Time taken: %.03f seconds, Fetched %d row(s).";
```

Comment:
Use `printInfo`. For distinguishing between rows and info rows.

Reply:
OK